arquillian-ape-sql/standalone/flyway/src/main/java/org/arquillian/ape/rdbms/flyway/FlywayOptions.java

```diff
@@ -0,0 +1,368 @@
+            flyway.setPlaceholderReplacement((Boolean) this.options.get(PLACEHOLDER_REPLACEMENT));
+        }
+
+        if (this.options.containsKey(PLACEHOLDERS)) {
+            flyway.setPlaceholders((Map<String, String>) this.options.get(PLACEHOLDERS));
+        }
+
+        if (this.options.containsKey(PLACEHOLDER_SUFFIX)) {
+            flyway.setPlaceholderSuffix((String) this.options.get(PLACEHOLDER_SUFFIX));
+        }
+
+        if (this.options.containsKey(PLACEHOLDER_PREFIX)) {
+            flyway.setPlaceholderPrefix((String) this.options.get(PLACEHOLDER_PREFIX));
+        }
+
+        if (this.options.containsKey(SQL_MIGRATION_PREFIX)) {
+            flyway.setSqlMigrationPrefix((String) this.options.get(SQL_MIGRATION_PREFIX));
+        }
+
+        if (this.options.containsKey(REPEATABLE_SQL_MIGRATION_PREFIX)) {
+            flyway.setRepeatableSqlMigrationPrefix((String) this.options.get(REPEATABLE_SQL_MIGRATION_PREFIX));
+        }
+
+        if (this.options.containsKey(SQL_MIGRATION_SEPARATOR)) {
+            flyway.setSqlMigrationSeparator((String) this.options.get(SQL_MIGRATION_SEPARATOR));
+        }
+
+        if (this.options.containsKey(BASELINE_VERSION)) {
+            flyway.setBaselineVersionAsString((String) this.options.get(BASELINE_VERSION));
+        }
+
+        if (this.options.containsKey(BASELINE_DESCRIPTION)) {
+            flyway.setBaselineDescription((String) this.options.get(BASELINE_DESCRIPTION));
+        }
+
+        if (this.options.containsKey(SQL_MIGRATION_SUFFIX)) {
+            flyway.setSqlMigrationSuffix((String) this.options.get(SQL_MIGRATION_SUFFIX));
+        }
+
+        if (this.options.containsKey(BASELINE_ON_MIGRATE)) {
+            flyway.setBaselineOnMigrate((Boolean) this.options.get(BASELINE_ON_MIGRATE));
+        }
+
+        if (this.options.containsKey(OUT_OF_ORDER)) {
+            flyway.setOutOfOrder((Boolean) this.options.get(OUT_OF_ORDER));
+        }
+
+        if (this.options.containsKey(SKIP_DEFAULT_CALLBACK)) {
+            flyway.setSkipDefaultCallbacks((Boolean) this.options.get(SKIP_DEFAULT_CALLBACK));
+        }
+
+        if (this.options.containsKey(SKIP_DEFAULT_RESOLVERS)) {
+            flyway.setSkipDefaultResolvers((Boolean) this.options.get(SKIP_DEFAULT_RESOLVERS));
+        }
+
+        if (this.options.containsKey(CALLBACKS)) {
+            flyway.setCallbacks((FlywayCallback[]) this.options.get(CALLBACKS));
+        }
+
+        if (this.options.containsKey(RESOLVERS)) {
+            flyway.setResolvers((MigrationResolver[]) this.options.get(RESOLVERS));
+        }
+    }
+
+    public static class FlywayConfigurationOptions {
+        private FlywayOptions flywayOptions = new FlywayOptions();
+
+        private FlywayConfigurationOptions() {
+        }
+
+        public FlywayConfigurationOptions installedBy(String installedBy) {
+            flywayOptions.put(FlywayOptions.INSTALLED_BY, installedBy);
```

Comment:
I know, but then we have to migrate all issues from JIRA and close the component for APE there + change website generation :)